mcp/setup.sh

```diff
@@ -3,12 +3,12 @@ set -o pipefail
 
 entry(){
   echo ""
-  echo "Multi-Cloud-Platform Setup"
+  echo "MCP Setup"
```

Comment:
Leave this as you had it and call the script mcpadm so that you don't have a command which is `setup setup`